sites/public/src/lib/hooks.ts

```diff
@@ -100,6 +101,24 @@ export async function fetchBaseListingData({
     if (orderDir) {
       params.orderDir = orderDir
     }
+
+    if (process.env.bloomJurisdictionNames.length != 0) {
+      const jurisdictions = await fetchBloomJurisdictionsByName()
```

Comment:
Am I reading correctly that every time `fetchBaseListingData` is called it requires another API call to the Bloom backend to map names to IDs?

Reply:
See other comment for more context -- it only gets called once, but through some TS magic that I don't quite understand via the `bloomJurisdictions` variable sticking around.

Reply:
Interesting... can we add a comment here to document this behavior?  I know you added a comment elsewhere, but it's not immediately obvious that it also affects this section of code

Reply:
Yes -- FWIW this is just me copying the same design pattern on the Bloom side via `fetchJurisdictionByName` but I am not super attached to it